cmd/neofs-adm/internal/modules/morph/group.go

```diff
@@ -0,0 +1,113 @@
+package morph
+
+import (
+	"encoding/json"
+	"fmt"
+	"path"
+
+	"github.com/nspcc-dev/neo-go/cli/input"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+	"github.com/spf13/viper"
+)
+
+const contractWalletName = "contract.json"
```

Comment:
To be more clear I'd name it
```suggestion
const contractWalletFilename = "contract.json"
```

Reply:
Fixed